$NetBSD: patch-aa,v 1.1 2000/08/29 20:48:13 jlam Exp $

Until the source catches up to the current Perl API, we need PERL_POLLUTE
to look like an older perl.

--- Makefile.PL.orig	2000-10-06 04:47:28.000000000 +0900
+++ Makefile.PL	2015-02-08 22:54:39.000000000 +0900
@@ -177,7 +177,7 @@ WriteMakefile(
 	      'INC'		=>	$my_includes,
 	      'LIBS'		=> 	[$lib_line],
 	      'MYEXTLIB'	=>	$my_extlib,
-	      'DEFINE'		=> 	"$v3_def $ssl_def",
+	      'DEFINE'		=> 	"$v3_def $ssl_def -DPERL_POLLUTE=1",
 	      'XSOPT'		=>	"-nolinenumbers",
 	      @extras
 );
